launch/state_machines/challenge_final.launch

```diff
@@ -13,18 +13,25 @@
 			<arg name="faces_db" value="~/MEGA/data/$(optenv ROBOT_ENV rgo2018)/models/openface_ros/faces.pkl" />
 		</include>
 
-		<!-- Joke -->
-		<include file="$(find robot_launch_files)/launch/essentials/joke.launch">
-			<arg name="machine" value="amigo2" />
-		</include>
+		<!-- Launch the websocket server (communication via roslib) -->
+		<node name="rosbridge_websocket_amigo2" pkg="rosbridge_server" type="rosbridge_websocket" output="log">
```

Comment:
This should be removed. Howevre, we first have to get rid of our `rosbridge_server` fork in order to remove this.

Reply:
Could you create an issue for this, because I have no idea about the whole rosbridge fork and what the difference is functionally

Reply:
I see that we are not ahead of the original repo. Only behind, a lot. What is the reason why we forked it?
@reinzor, @Rayman

Reply:
Is that issue https://github.com/tue-robotics/tue-env/issues/200?

Reply:
Now that tue-robotics/tue-env#200 is solved. What should be done now?

Reply:
Ping @reinzor

Reply:
Now one one rosbridge should suffice